EventFilter/HcalRawToDigi/plugins/PackerHelp.h

```diff
@@ -587,4 +627,44 @@ class UHTRpacker {
   };
 };
 
+// converts HE QIE digies to HB data format
+
+QIE11DataFrame convertHB2HE(QIE11DataFrame qiehe) {
+  QIE11DataFrame qiehb = qiehe;
+  int adc_, tdc_, capid_;
```

Comment:
Normally a trailing underscore is reserved to class members, not for local variables of a class method

Reply:
renamed these local variables in commit 768d39c